### PR TITLE
Update people

### PR DESCRIPTION
**Pull request by tae898**

### CODE REVIEW

File: people/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
----
-title: People
----
+______________________________________________________________________
```
Comment (miselico) [inline people/index.md]: We have to keep the markdown block as is. The block between --- contains information that is parsed by jekyll to indicate that this must be rendered as a page

File: people/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-# Group Members #
+## title: People
+
+# Group Members
 
 <div class="people_gallery">
-    <figure><img src="{{ site.baseurl }}images/members/Andreas.png" class="people_gallery__img" alt="Image 1">
+    <figure><a href="http://sauter.at/"><img src="{{ site.baseurl }}images/members/Andreas.png" class="people_gallery__img" alt="Image 1">
```
Comment (miselico) [inline people/index.md:8]: For all links, you are missing the closing of the tag. So, before the img, you open it with <a ...> after the caption, you must close it with </a>

**Review by miselico — CHANGES_REQUESTED** [review]
Can you revert the front matter (the part between the ---'s). And then also add the closing tags for the links?

**Review by miselico — APPROVED** [review]
LGTM

**Comment by tae898** [comment]
image

**Pull request merged**